### PR TITLE
Change madlib gppkg  version string

### PR DESCRIPTION
**Pull request by jingyimei**
This commit changes the naming convention for madlib gppkg,
after renaming, the format of madlib gppkg will look like:
madlib-1.13_dev-gp5-rhel6-x86_64.gppkg
madlib-1.12_dev-gp4.3orca-rhel5-x86_64.gppkg

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/297/

**Comment by fmcquillan99** [comment]
naming convention looks good.

I assume for prod releases there is no "_dev" in the name.

**Comment by jingyimei** [comment]
@fmcquillan99 Yes, the version string is saved in Version.yml file and when we make the release, we will change the version string from 1.13_dev to 1.13, and cmake will directly get it from the yml file.

File: cmake/LinuxUtils.cmake [hunk 1]
```diff
@@ -9,3 +9,14 @@ macro(rh_version OUT_VERSION)
         set(${OUT_VERSION} "${OUT_VERSION}-NOTFOUND")
     endif(EXISTS "/etc/redhat-release")
 endmacro(rh_version)
+
```
[the remark this answers is not in the document]
Comment (jingyimei) [reply cmake/LinuxUtils.cmake]: yes, I can use a regex to grep RH_MAJOR_VERSION from RH_VERSION after running`rh_version(RH_VERSION)`.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/302/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/303/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/308/

**Pull request closed without merging**